v1.6.0.md

```diff
@@ -0,0 +1,487 @@
+- [backport v1.6] [BUG] Can't add volume, delete and backup vm when vm enabled cpu pinning while cpu manager is `disabled` [#8834](https://github.com/harvester/harvester/issues/8834)
+- [BUG] Can't add volume, delete and backup vm when vm enabled cpu pinning while cpu manager is `disabled` [#8778](https://github.com/harvester/harvester/issues/8778)
+- [BUG] ip count on subnet not managed correctly when VMs are stopped and deleted [#8693](https://github.com/harvester/harvester/issues/8693)
+
+### Known Issues of external Rancher
+
+- [BUG] Unable to select upgrade version on Harvester upgrade dashboard from Rancher v2.12.0 [#8952](https://github.com/harvester/harvester/issues/8952)
+- [BUG] Can't add member to project in Harvester v1.6.0-rc6 on Rancher v2.12.0 with UI extension v1.6.0-rc6 [#8942](https://github.com/harvester/harvester/issues/8942)
+- [BUG] Rancher v2.12.0 w/ Harvester v1.6.0-rc6 w/ Harvester UI Extension v1.6.0-rc6 Can't Add Cluster Members to imported Harvester Cluster [#8923](https://github.com/harvester/harvester/issues/8923)
+- [BUG] Can't add volume to vm in Harvester from Rancher v2.12.0 integrate mode, available volume is hidden beneath the panel #[8885](https://github.com/harvester/harvester/issues/8885)
+- [BUG] Can't migrate vm in Harvester from Rancher v2.12.0 integrate mode, no available target node in the list [#8884](https://github.com/harvester/harvester/issues/8884)
+- [BUG] Unable to move namespace to another project [#8851](https://github.com/harvester/harvester/issues/8851)
+
+## Deprecations and Removals 
+
+### Rancher Kubernetes Engine (RKE)
+
+RKE reached its end-of-life on July 31, 2025. Because of this, support for RKE has been removed from Harvester v1.6.0. You will no longer be able to run Rancher-provisioned RKE clusters on this version or any future versions.
+
+For better security and efficiency, you may want to replatform to RKE2. This process involves [creating new RKE2 clusters](https://docs.harvesterhci.io/v1.6/rancher/node/rke2-cluster) and migrating existing workloads from the old RKE clusters. In-place migration from RKE to RKE2 is not an option because of possible data loss. For more information, see [RKE End of Life](https://www.suse.com/support/kb/doc/?id=000021513).
+
+### `install.wipedisks` setting
+
+This setting was removed from Harvester v1.5. If you want to clear disk partitions with the `COS_OEM` label, use [`install.wipe_all_disks`](https://docs.harvesterhci.io/v1.6/install/harvester-configuration#installwipe_all_disks) and [`install.wipe_disks_list`](https://docs.harvesterhci.io/v1.6/install/harvester-configuration#installwipe_disks_list) instead.
+
+## Components
+
+| Component | Version |
+| --- | --- |
+| CDI | v1.62.0 |
+| KubeVirt | v1.5.2 |
+| Longhorn | v1.9.1 |
+| Rancher (embedded) | v2.12.0 |
+| RKE2 | v1.33.3 |
+| SLE Micro | 5.5 |
```

Comment:
The component versions are missing links off to the component releases. Compare with v1.5.1:

https://github.com/harvester/release-notes/blob/d6e72ef6d897bb6044ae38c16c364399ddb73b09/v1.5.1.md?plain=1#L128-L134

These links should have been auto-generated when running `harvester-release ... release-changelog`

Reply:
nice catch, updated